src/main/java/me/marcellin/timechecker/api/v1/time/TimeController.java

```diff
@@ -16,13 +19,14 @@
     @Autowired
     private TimeService timeService;
 
-    @RequestMapping("/{queryCity}")
-    public TimeModel check(
+    @RequestMapping(value = "/{queryCity}", method = RequestMethod.GET,  produces = "application/json")
```

Comment:
forgot to add the link to where queryCity is being used
https://github.com/nshimiye/time-checker/pull/2/files/bf25ff85ff74980c6070e35d8e7e07beecb7e638#diff-7a5f9c3062a22b93b035c04266c85fbfR24